tezster.js

```diff
@@ -394,13 +427,12 @@ program
 });
 
 program
-.command('help')
-.action(async function(){
-    const tezsterManager = require('./tezster-manager');
-    console.log(tezsterManager.helpData); //'\x1b[33m%s\x1b[0m',         
+.on("--help", () => {
+  console.log();
+  console.log("To know more about particular command usage:");
+  console.log("\ttezster [command] --help");
```

Comment:
Combine all log statement, use JS 'template' notation and use \n to add new line if required.